<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="startButton_a">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stopButton_a">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="resetButton_a">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QProgressBar" name="progressBar_a">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="startButton_b">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="stopButton_b">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="resetButton_b">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QProgressBar" name="progressBar_b">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="startButton_c">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="stopButton_c">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="resetButton_c">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QProgressBar" name="progressBar_c">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
